README.md

```diff
@@ -17,3 +17,31 @@ Within this container, use the provided user maintenance command-line utility to
 To deploy the STT Docker container, run the `deploy.sh` script on the host server.
 
     ssh -i PRIVATE_KEY_PATH USERNAME@HOSTNAME 'bash -s' < deploy.sh
+
+## Setting up a new environment
+
+### Cloudformation
+There is a cloudformation-prod.json file in the repo root.  This is a cloudformation template.  All you need to do is login to AWS and go to the cloudformation service.  You will then create a new stack and give it this file.  When you create the stack it will ask you for all the credentials for this new deployment.  So beforehand you must create username and password for the db, STT, and ODK.  After this template is run and the stack is created there will be an ec2 instance, RDS mysql db, and security groups to go along.
```

Comment:
This is great! Thanks for getting the ball rolling on updated documentation.

On a side note, do you have any idea if it's possible to be more flexible in our CloudFoundation config with the region in which is stack is deployed? (possibly something relevant here -- http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/pseudo-parameter-reference.html)

Barring an enhancement to support any region / availability zone, that AWS region restriction should be added to the README.